buddyboss/buddyboss-platform
# "Heading" Profile search field issue

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #347 .

### How to test the changes in this Pull Request:

1. Go to 'Admin Dashboard >> BuddyBooss >> Profile >> Profile Search'
2. Add the Heading type field.
3. Again try to Add the second Heading type field.
4. Now click on the Update button. You will see the second heading type field is disappeared.

### Proof Screenshots or Video
https://www.loom.com/share/accfa6e328524e08976c1be87e085dbf



### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
